### PR TITLE
Add key to remove error

### LINKED ISSUE #13: Warning:  Each child in a list should have a unique "key" prop.
Upon installing the plugin, I started getting warnings in the console:

```
Warning: Each child in a list should have a unique "key" prop.

Check the top-level render call using <head>. See https://fb.me/react-warning-keys for more information.
    in meta
```

Not sure if it's important to be fixed. Will appreciate your attention.

### PR DESCRIPTION
**Pull request by Primajin**
fixes #13

```

 ERROR

Warning: Each child in a list should have a unique "key" prop.

Check the top-level render call using <head>. See https://reactjs.org/link/warning-keys for more information.
    at meta
    at HTML (R:\eyesbound\.cache\page-ssr\routes\render-page.js
```

### CODE REVIEW

**Review by w3bdesign — APPROVED** [review]

**Comment by Primajin** [comment]
Thanks @w3bdesign for approving the changes. Do you think we can merge this soon~ish?

**Comment by w3bdesign** [comment]
> Thanks @w3bdesign for approving the changes. Do you think we can merge this soon~ish?

Yes, we can merge.

Just need approval from someone with write access.
@Primajin

**Review by w3bdesign — CHANGES_REQUESTED** [review]
Would it be better to use import instead of require, and const instead of let?

**Comment by w3bdesign** [comment]
@Primajin

**Comment by Primajin** [comment]
I am happy helping transforming this from common JS into a module (ESM `import` vs. CJS `require`).
However this could be considered a breaking change and I highly recommend doing this in a separate PR with a proper plan and discussion 😅

Let's keep the diff of this PR here small as it's sole purpose was fixing the error with the missing key, happy to file other PRs for chore (`let` vs. `const` etc.) if applicable.

**Review by w3bdesign — APPROVED** [review]

**Review by w3bdesign — APPROVED** [review]

**Comment by Primajin** [comment]
Hey @thomkrupa @sweaverdeloitte do you have write access to this repo? This is a pretty small diff and many people are affected by the bug, can we go forward with this? Would be great, thank you!

**Review by thomkrupa — APPROVED** [review]

**Comment by thomkrupa** [comment]
Thank you @Primajin! Merged. We're planning a bigger update to the plugin soon!

**Comment by Primajin** [comment]
Sweet, looking forward to. Let me know if you need any support

**Pull request merged**
